docs/src/modules/java/pages/timers.adoc

```diff
@@ -0,0 +1,152 @@
+= Timers
+:page-supergroup-java-scala: Language
+
+Timers allow for scheduling calls in the future. For example, to verify that some process have been completed or not.
+
+Timers are persisted in the https://docs.kalix.io/reference/glossary.html#proxy[Kalix proxy] and are guaranteed to run at least once. 
+
+When a timer is triggered, the scheduled call is executed. If successfully excuted, the timer completes and is automatically removed. In case of a failure, the timer is rescheduled to run once more after 3 seconds. This process repeats until the call succeeds.
```

Comment:
The 3 seconds backoff is hard-coded in the config, but that is super arbitrary. We can go with this default for now (or change it), but we should consider adding it as part of the timer registration.

Reply:
To me "once more" reads like it only repeats once which then the next paragraph argues with, I know you love a good argument but maybe replace that and the next section with "this repeats every 3 seconds until the call succeeds"